source/hook.cpp

```diff
@@ -111,7 +111,7 @@ struct ThreadData {
 static bool isKeyDown(key_t k) {
 #if defined(_WIN32)
 	// Windows is incredibly simple, this allows us to query a key without any hooks.
-	return !!GetAsyncKeyState(k);
+	return GetAsyncKeyState(k) >> 15;
```

Comment:
Missing explicit conversion to `bool`, add `(bool)` or `!!` in front of the `(GetAsyncKeyState(k) >> 15)` call.

Reply:
Done.